app/shares.php

```diff
@@ -134,14 +102,14 @@
 		'form' => 'simple',
 		'method' => 'GET',
 	),
-	'lemmy' => array(
-		'url' => '~URL~/create_post?url=~LINK~&name=~TITLE~',
+	'pocket' => array(
+		'url' => 'https://getpocket.com/save?url=~LINK~&amp;title=~TITLE~',
```

Comment:
Is this changing it from set your own server URL to a specific one? Mozilla's working on opening up the server (cf. `https://github.com/Pocket/extension-save-to-pocket/issues/75#issuecomment-655311178`) and there's also https://github.com/open-pocket/open-pocket

It might be worth taking into consideration somehow, but no big deal for now.

Reply:
This div does not make sense.
The pocket settings I did not change.
But after all I changed the order of the array, that is why it looks weird

Reply:
reordering made with commit e7ca768

Reply:
I see, it says Lemmy. Never mind.